Jellyfin.Api/Controllers/MediaInfoController.cs

```diff
@@ -183,9 +202,9 @@ public async Task<ActionResult<PlaybackInfoResponse>> GetPlaybackInfo([FromRoute
                         new LiveStreamRequest
                         {
                             AudioStreamIndex = audioStreamIndex,
-                            DeviceProfile = deviceProfile?.DeviceProfile,
-                            EnableDirectPlay = enableDirectPlay,
-                            EnableDirectStream = enableDirectStream,
+                            DeviceProfile = playbackInfoDto?.DeviceProfile?.DeviceProfile,
```

Comment:
:point_up:

Reply:
It does, but only because it was migrated incorrectly